llvm/lib/Target/X86/X86InstrMisc.td

```diff
@@ -1223,7 +1223,7 @@ multiclass Bls<string m, Format RegMRM, Format MemMRM, X86TypeInfo t, string Suf
                              (outs t.RegClass:$dst), []>, T8, VVVV;
 }
```

Comment:
I'm afraid `NoCD8` is missing again. I'm very worried about the bug that the missing `NoCD8` causes encoding errors. Could you create a NFC PR that adds NoCD8 to class ITy, and rewrite all the promoted instructions with `ITy`?